Automattic/wp-calypso
# Domain transfer: Add animated dots

## Proposed Changes

Add a simple animation on the “…” when validating the domain. The idea is to tell the user that we are handling it.

https://github.com/Automattic/wp-calypso/assets/2653810/d25f7cd9-6c62-42a0-92d0-075b2173cb3f



## Testing Instructions

If it helps, change your browser to use slow 3g.

- Pull and run this branch or use calypso live link
- Navigate to `/setup/domain-transfer/domains`
- Add a domain and fake code (for testing)
- You should notice the dots animating while validating the domain/code
